.gitignore

```diff
@@ -4,6 +4,8 @@
 /site/partials/recipes.html
 /site/docs/
 
+yarn-error.log
```

Comment:
could you please use package-lock.json instead? I think this project doesn't use yarn

Reply:
You think? I'll remove yarn.lock and add it to gitignore